core/quantity/src/Formatter/Format.ts

```diff
@@ -107,19 +109,53 @@ export class Format {
     }
   }
 
+  public static isFormatTraitSetInProps(formatProps: FormatProps, trait: FormatTraits) {
+    if (!formatProps.formatTraits)
+      return false;
+    const formatTraits = Array.isArray(formatProps.formatTraits) ? formatProps.formatTraits : formatProps.formatTraits.split(/,|;|\|/);
+    const traitStr = Format.getTraitString(trait);
+    return formatTraits.find((traitEntry) => traitStr === traitEntry) ? true : false;
+  }
+
+  /** Get string used in FormatProps  */
+  public static getTraitString(trait: FormatTraits) {
```

Comment:
@bsteinbk can you add some tests for this?  There's a bunch of similar ones in ecschema-metadata in the `test/ParseUtils.test.ts`

Reply:
Test added... and with it I found one missing case - thanks!